phpBB/assets/javascript/update_worker.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+function webpushWorkerUpdate() {
+	if ('serviceWorker' in navigator) {
+		navigator.serviceWorker.getRegistration(serviceWorkerUrl)
+			.then(registration => {
+				registration.update();
+			})
+			.catch(error => {
+				// Service worker could not be updated
+				console.info(error);
+			});
+	}
+}
+
+// Do not redeclare function if exist
+if (typeof domReady === 'undefined') {
+	window.domReady = function(callBack) {
+		if (document.readyState === 'loading') {
+			document.addEventListener('DOMContentLoaded', callBack);
+		} else {
+			callBack();
+		}
+	};
+}
+
+/* global domReady */
```

Comment:
@marc1706 do you know why stuff like this `domReady` failed linting here w/o having to make that global content, but when you had the same code in your webpush.js file, that passed linting no problem?

Reply:
![magic](https://github.com/phpbb/phpbb/assets/394418/0646c2f4-2a12-4b42-a50d-3894c4df564b)